### PR TITLE
[FlashScript] add instructions for RGBW2 device.

### PR DESCRIPTION
**Pull request by andyblac**
take a look, if OK with you, please pull.

I'll report back once Shelly has responded to my support ticket regarding flash modes.

Thanks
Andy.

### CODE REVIEW

File: tools/flash_shelly.sh [hunk 1]
```diff
@@ -139,7 +139,18 @@ function write_flash {
...
+          echo " "
+          echo "To finalise flash process you will need to switch 'Modes' in the device WebUI,"
+          echo "WARNING!! If you are using this device in conjunction with Homebridge it will"
+          echo "result in ALL scenes / automations to be removed within HomeKit."
+          echo "Goto http://$device in your web browser"
+          echo "Goto settings section"
+          echo "Goto 'Device Type' and switch modes"
+          echo "Once mode has been changed, you can switch it back to your preferred mode."
+        else
         echo "Flash failed!!!"
```
Comment (timoschilling) [inline tools/flash_shelly.sh:152]:
```suggestion
          echo "Flash failed!!!"
```

Comment (andyblac) [reply tools/flash_shelly.sh:152]: oops, good catch, thx

**Comment by rojer** [comment]
merged, thanks!

**Comment by andyblac** [comment]
@rojer just to keep you updated, I'm waiting for a call from Shelly regarding the issue with RGBW2 flashing.

**Pull request merged**